config/routes.rb

```diff
@@ -66,7 +66,6 @@
                                                     :month  => /\d{2}/,
                                                     :day    => /\d{2}/
 
-  get "/blog.rss" => "blog#feed"
```

Comment:
I think we also need to remove code from the blog_controller.rb

Reply:
Good call, just pushed a new commit.